infrahub_sdk/pytest_plugin/items/python_transform.py

```diff
@@ -36,9 +36,11 @@ def instantiate_transform(self) -> None:
         relative_path = (
             str(self.resource_config.file_path.parent) if self.resource_config.file_path.parent != Path() else None  # type: ignore[attr-defined]
         )
-        self.transform_instance = self.resource_config.load_class(  # type: ignore[attr-defined]
+        transform_class = self.resource_config.load_class(  # type: ignore[attr-defined]
             import_root=self.repository_base, relative_path=relative_path
         )
+        client = self.session.infrahub_client  # type: ignore[attr-defined]
+        self.transform_instance = transform_class(branch=client.default_branch, client=client)
```

Comment:
A transform instance was formerly instantiated using `branch=""` (through `get_transform_class_instance` default parameter). I am not sure how transform class uses this parameter afterwards, but we may want to keep the same here?

Reply:
Yes, we can keep that behavior and I will set it as a blank string.